test/server/fixtures/field.html.ejs

```diff
@@ -23,9 +22,15 @@
       var recurly = new parent.recurly.Recurly(recurlyConfig());
       var inputs = body.inputs;
       var id = body.id;
+      var frame;
+
+      for (var i = 0; i < window.parent.frames.length; i++) {
+        frame = window.parent.frames[i].contentWindow || window.parent.frames[i];
+        if (frame === undefined) window.parent.frames.splice(i, 1);
+      }
```

Comment:
Is the intent of this to clear a relay frame that's been abandoned? If so, the spec causing it needs to be fixed so that this isn't necessary

Reply:
I'll narrow down the spec that's causing this to break, update the spec and update this .ejs file